### PR TITLE
Add basic testing

### PR DESCRIPTION
**Pull request by bowenwang1996**
Add some basic testing for base64 encoding using [as-pect](https://github.com/jtenner/as-pect).

### CODE REVIEW

File: assembly/__tests__/as-pect.d.ts [hunk 1]
```diff
@@ -0,0 +1,483 @@
+/**
```
Comment (vgrichina) [inline assembly/__tests__/as-pect.d.ts]: can we avoid committing this into repo? Like e.g. copy from `node_modules` when running the tests.

File: assembly/__tests__/near.spec.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import { base64 } from '../../near';
+
+function stringToBytes(s: string): Uint8Array {
```
Comment (vgrichina) [inline assembly/__tests__/near.spec.ts]: there is `stringToBytes` and `bytesToString` already in `near` https://github.com/nearprotocol/near-runtime-ts/blob/master/near.ts#L1096

File: assembly/__tests__/as-pect.d.ts (hunk 1 above)
Comment (bowenwang1996) [reply assembly/__tests__/as-pect.d.ts]: Fixed

File: assembly/__tests__/near.spec.ts (hunk 2 above)
Comment (bowenwang1996) [reply assembly/__tests__/near.spec.ts]: Fixed

**Pull request merged**